internal/server/routes.go

```diff
@@ -36,7 +36,7 @@ func InitMux(scrapeServer *scrapeServer) (*http.ServeMux, error) {
 	mux.HandleFunc("GET /extract/headless", h)
 	mux.HandleFunc("POST /extract/headless", h)
 	mux.HandleFunc("POST /batch", scrapeServer.batchHandler())
-	mux.HandleFunc("DELETE /{$}", scrapeServer.deleteHandler())
+	mux.HandleFunc("DELETE /extract", scrapeServer.deleteHandler())
```

Comment:
Just realized that the / path was not really right. This endpoint deletes single urls from the db, so it's really part of `extract`.